pkg/issuer/acme/dns/dnsimple/dnsimple_test.go

```diff
@@ -0,0 +1,50 @@
+package dnsimple
```

Comment:
These tests will pass, I have yet been able to get this running in minikube successfully